test/e2e/egress_test.go

```diff
@@ -28,15 +28,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
+	k8sUtilNet "k8s.io/utils/net"
```

Comment:
package name and alias should use lowercases. In other places its aliass is "utilnet"

Reply:
Done